### ISSUE TITLE
Address label and contrast issues in work edit form

### ISSUE THREAD

**Issue by awead** [psu-stewardship/scholarsphere-4#475]
This is a sub-ticket of #416 and the details can be found there.

## WDLL
* correct spelling of separate ✅ 
* "x" in keywords should be labeled delete or remove ✅ 
* the work type dropdown does not appear to the screen reader ✅ 
* "+" and "-" signs need to be labeled as add and delete ✅ 
* creators sub-heading under "Contributors" needs higher contrast ✅ 
* screen reader cannot see the license dropdown in the publish tab (see #514)
* radio buttons for visibility need to be labeled ✅ 
* deposit agreement checkbox needs a label ✅ 
* multiple links with "see more" -- these should have different text to set them apart ✅
